### DISCUSSION TITLE
Page title not refresh when keepalive is enable.

### DISCUSSION THREAD

**Discussion by kiki-kanri** [nuxt/nuxt#18660] [Questions]
When nuxt config keepalive is enable, go to page A, page B and then page A (using NuxtLink), page title not refresh to A page's title.

The way to set the page title is use Head and Title component.

**Comment by ndragun92** [comment]
I assume it is because it is a wrapper inside keep-alive. Did you try to use it via useHead inside the script to see if the same happens?

**Reply by kiki-kanri** [reply]
useHead() in script is the same as using Head and Title components, but use onActivated and useHead is works.

Examples:
```
<script setup>
useHead({
    title: 'title'
});
</script>
```
Is same result.

```
onActivated(() => {
    useHead({
        title: 'title'
    });
});
```
Is works.
